configuration/pih/drugs/drugs.csv

```diff
@@ -422,77 +477,80 @@ c6a94776-fce4-11e9-8f0b-362b9e155667,18228,"Rifampicin (R) 75mg + Isoniazid (H)
+30016839-8b3d-4eea-a647-0d8342e44ddb,12129,"Thiamine (Vitamin B1), 50mg tablet",CIEL:84995,
 35816ccb-904b-4ca0-8160-d7de356d5f1e,11801,"Thiamine (Vitamin B1), Solution for injection, 50mg/mL, 2mL ampoule",CIEL:84995,
-88c829e2-081b-4082-9185-79928a39f305,10219,"Thiopental, Powder for solution for injection, 500mg vial",CIEL:85015,
-25da272a-5892-4c53-87bd-96c6a2d8689f,11029,"Timolol maleate, Eye drop, 0.25%, 5mL bottle",CIEL:85134,
-48bad836-e456-40db-be79-769e2798b23e,11121,"Tramadol hydrochloride, 50mg capsule",CIEL:85276,
-43271fad-6f01-4a94-9e4f-1746642a3af6,10501,"Tramadol hydrochloride, Solution for injection, 50mg/mL, 2mL ampoule",CIEL:85276,
-8319e5d6-777e-4d0e-9aad-099e684dcc8e,11555,"Tranexamic acid, Solution for injection, 100mg/mL, 5mL ampoule",CIEL:85286,
+88c829e2-081b-4082-9185-79928a39f305,11069,"Thiopental, Powder for solution for injection, 500mg vial",CIEL:85015,
+25da272a-5892-4c53-87bd-96c6a2d8689f,10766,"Timolol maleate, Eye drop, 0.25%, 5mL bottle",CIEL:85134,
+48bad836-e456-40db-be79-769e2798b23e,10165,"Tramadol hydrochloride, 50mg capsule",CIEL:85276,
+43271fad-6f01-4a94-9e4f-1746642a3af6,10034,"Tramadol hydrochloride, Solution for injection, 50mg/mL, 2mL ampoule",CIEL:85276,
+8319e5d6-777e-4d0e-9aad-099e684dcc8e,11309,"Tranexamic acid, Solution for injection, 100mg/mL, 5mL ampoule",CIEL:85286,
 a40d9d90-c981-11e7-abc4-cec278b6b50a,11665,"Tretinoin, Skin cream, 0.05%, 20 gram tube",CIEL:85330,
+678d2718-abe3-4c7d-bd6d-e1761d9a5eeb,11393,"Trihexyphenidyl, 2mg tablet",PIH:915,
 c18e751c-1ec5-420b-8ba7-68a22c5d1acc,10660,"Trihexyphenidyl, 5mg tablet",PIH:915,
 72a1bbde-5946-40d5-8750-918a47be523f,10647,"Tropicamide, Eye drop, 0.5%, 5mL bottle",CIEL:85597,
-af8c4eaa-dfbe-11e9-8a34-2a2ae2dbcce4,11069,"Tuberculin purified protean derivative (PPD), 5 Tuberculin Units (TU)/0.1mL, 1mL vial of 10 doses",CIEL:82886,
+af8c4eaa-dfbe-11e9-8a34-2a2ae2dbcce4,10996,"Tuberculin purified protean derivative (PPD), 5 Tuberculin Units (TU)/0.1mL, 1mL vial of 10 doses",CIEL:82886,
 af8c4fea-dfbe-11e9-8a34-2a2ae2dbcce4,11116,"Vaccine, Haemophilus Influenzae Type B (Hib) conjugate, monodose vial with solvent",CIEL:5261,
 78fc557e-dfbe-11e9-8a34-2a2ae2dbcce4,11157,"Vaccine, Meningitis (A+C+W135+Y) conjugate, monodose pre-filled syringe with solvent",CIEL:79549,
-af8c4d2e-dfbe-11e9-8a34-2a2ae2dbcce4,10766,"Vaccine, Pneumococcal conjugate, PCV13, 0.5mL monodose pre-filed syringe (Child)",CIEL:165575,
+af8c4d2e-dfbe-11e9-8a34-2a2ae2dbcce4,11433,"Vaccine, Pneumococcal conjugate, PCV13, 0.5mL monodose pre-filed syringe (Child)",CIEL:165575,
 af8c47ca-dfbe-11e9-8a34-2a2ae2dbcce4,10087,"Vaccine, Pneumococcal polysaccharide, PPSV23, 0.5mL monodose pre-filled syringe (Adult)",CIEL:82213,
-24e44864-7ab3-4919-8e75-004b838ae06c,10205,"Vaccine, Rabies, on vero cells, monodose vial, with diluent",PIH:413,
-78fb8112-dfbe-11e9-8a34-2a2ae2dbcce4,10165,"Vancomycin hydrochloride, Powder for solution for infusion, 1 gram vial",CIEL:86083,
```

Comment:
This seems correct.  https://obnav.pih-emr.org/openboxes/inventoryItem/showStockCard/10165